### PR TITLE
Avoid empty resource names in Yii in case of root `/` paths 

### PR DESCRIPTION
**Pull request by labbati**
### Description

Before, root paths `/` would result in empty resource names. Since in case of root path Yii defaults to `index`, we adopt this behavior while naming resources.

See:
  - https://github.com/yiisoft/yii2/blob/ff0fd9a9ea043bcd915f055a868585b945399864/framework/base/Controller.php#L245
  - https://github.com/yiisoft/yii2/blob/ff0fd9a9ea043bcd915f055a868585b945399864/framework/base/Controller.php#L55

Note: static analysis failure has been fixed [here](https://github.com/DataDog/dd-trace-php/pull/1121/files#diff-7d851e81db47080c3bc2bb779e04fb84559a9b83f44a0917c0bbe529490bd9e0R218)

### Readiness checklist
- [x] (only for Members) Changelog has been added to the release document.
- [x] Tests added for this feature/bug.

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.
- [ ] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: src/DDTrace/Integrations/Yii/YiiIntegration.php [hunk 1]
```diff
@@ -151,4 +151,23 @@ function (SpanData $span, $args) use ($service) {
...
+    /**
+     * Returns the resource name if it can be detected from the invocation args, null otherwise.
+     *
+     * @param array invocation params of teh Module and Controller::runAction method.
+     * @return string|null the obtained resource name or null if it cannot be detected.
+     */
+    public static function extractResourceNameFromRunAction($args)
+    {
+
+        if (isset($args[0]) && is_string($args[0])) {
```
Comment (morrisonlevi) [inline src/DDTrace/Integrations/Yii/YiiIntegration.php:164]: Minor suggestion here to use a fully qualified funtion name.
```suggestion
        if (isset($args[0]) && \is_string($args[0])) {
```

**Review by morrisonlevi — APPROVED** [review]

**Comment by labbati** [comment]
Static Analysis has been fixed in `master` via #1021

**Review by morrisonlevi — APPROVED** [review]
Approved, but I would slightly happier if you merged the latest master into this branch, or alternatively rebased this onto the latest master.

**Comment by labbati** [comment]
Thank you @morrisonlevi! I pulled `master` and it is all green now.

**Pull request merged**